js/context/UserContext/UserProvider.js

```diff
@@ -3,7 +3,12 @@ import {
   queryAddUserToken,
   queryRemoveUserToken,
   queryUpdateUser,
-  queryGetUser
+  queryGetUser,
+  queryAddActivity,
+  queryRemoveActivity,
+  queryAddJournal,
+  queryRemoveJournal,
+  queryFirstTime
```

Comment:
name this better.

Reply:
Changed queryFirstTime to 2 different queries named queryToggleShowOnBoarding and queryToggleShowPrivacyPolicy. Changed queryAddJournal and queryRemoveJournal to  queryAddJournalEntry and  queryAddJournalEntry